### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by liliia-kostenko**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,53 @@
...
+            average_rating: int,
+            count_of_ratings: int
+    ) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> None:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:28]: cars list of what?
Provide inner type

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,53 @@
...
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(income, 1)
+
+    def calculate_washing_price(self, car: list) -> None:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:36]: car is not list. It is instance of Car

File: app/main.py [hunk 3]
```diff
@@ -1,8 +1,53 @@
...
+        return round(
+            car.comfort_class
+            * (self.clean_power - car.clean_mark)
+            * self.average_rating
+            / self.distance_from_city_center,
+            1
+        )
+
+    def wash_single_car(self, car: None) -> None:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:45]: car is not None

File: app/main.py [hunk 4]
```diff
@@ -1,8 +1,53 @@
...
+        if self.clean_power > car.clean_mark:
+            car.clean_mark = self.clean_power
+
+    def rate_service(self, rating: None) -> None:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:49]: rating is not None. Wrong type anotations

File: app/main.py (hunk 2 above)
Comment (ArsenPidhoretskyi) [inline app/main.py:36]: Also wrong return type annotations. It returns float

File: app/main.py (hunk 1 above)
Comment (ArsenPidhoretskyi) [reply app/main.py:28]: Also wrong return type annotations

File: app/main.py [hunk 5]
```diff
@@ -1,8 +1,53 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str) \
+            -> None:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:7]:
```suggestion
            brand: str
        ) -> None:
```

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Comment by liliia-kostenko** [comment]
@ArsenPidhoretskyi

File: app/main.py (hunk 5 above)
Comment (vasylhnatiuk) [reply app/main.py:7]: do it

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

**Review by vsmutok — CHANGES_REQUESTED** [review]
Please fix tests

**Review by ArsenPidhoretskyi — APPROVED** [review]
Well done!

**Pull request closed without merging**